### ISSUE TITLE
Cloning repository loads non-stop

### ISSUE THREAD

**Issue by bcrhbrhcdb** [stackblitz/core#2990]
### **Description of Bug**
Provide a concise description of your bug and your project link (if applicable).
https://stackblitz.com/~/github.com/bcrhbrhcdb/my
Whenever i try to open a project, it never loads. The 'Cloning from main branch' message stays, but nothing happens.
### **Expected Behavior**
Everything to load, exstensions, files, ect.

### **Screenshots/Screencast**
[Screen recording 2024-02-28 14.22.24.webm](https://github.com/stackblitz/core/assets/123518949/379db8df-eed7-4b02-bfdf-55c25040e50d)

**Additional Context/Questions**
Add any other context or questions regarding this bug.

**Comment by fvsch** [comment]
Hi @bcrhbrhcdb.

In your screen recording you show that StackBlitz gets stuck trying to clone the `main` branch of your GitHub repository `bcrhbrhcdb/my`.

What I don't understand is that your screen recording was posted a couple years ago, and the only visible commit on the `main` branch of that repository was pushed roughly 20 minutes earlier, but in your screen recording you typed “been like this for 21 hours”.

Has that repository existed for more than a few hours?
And was it private when you were initially trying to load it in StackBlitz and made that screen recording?

**Comment by coder0107git** [comment]
I'm running into the same issue with one of my private repos. I ran into the same issue Monday or Tuesday but I was able to resolve it then (I can't remember what I did to make it work again unfortunately).

**Comment by coder0107git** [comment]
@bcrhbrhcdb What do you see when you open [webcontainer.io](https://webcontainer.io/)? (i.e. is it blocked?)

**Comment by fvsch** [comment]
It would be interesting as well to try to load your repo in the new StackBlitz editor and report back any error in the browser's Console.

**Comment by bcrhbrhcdb** [comment]
Also, it was recorded on 2/28/24.

**Comment by bcrhbrhcdb** [comment]
Here is a screen recording of whats not working: 
Also, webcontainer.io is not blocked. Can you by any chance send me the new stackblizt editor? 

https://github.com/stackblitz/core/assets/123518949/0157dc13-6aab-4a27-a43a-08b88ca2d998

**Comment by bcrhbrhcdb** [comment]
My console opening a project: 
[Screen recording 2024-02-29 08.58.53.webm](https://github.com/stackblitz/core/assets/123518949/53eb4387-8667-4549-8b87-ab0d32809dec)

**Comment by bcrhbrhcdb** [comment]
Maybe its something with the link? 
https://stackblitz.com/~/github.com/bcrhbrhcdb/ef

**Comment by bcrhbrhcdb** [comment]
![Screenshot 2024-02-29 10 15 56](https://github.com/stackblitz/core/assets/123518949/55d655cc-83b6-4178-925f-74181c950d29)

**Comment by bcrhbrhcdb** [comment]
The bug also happens with all of the other repos, even if they aren't mine...

**Comment by coder0107git** [comment]
> ![Screenshot 2024-02-29 10 15 56](https://private-user-images.githubusercontent.com/123518949/308995307-55d655cc-83b6-4178-925f-74181c950d29.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDkyNDE3MDMsIm5iZiI6MTcwOTI0MTQwMywicGF0aCI6Ii8xMjM1MTg5NDkvMzA4OTk1MzA3LTU1ZDY1NWNjLTgzYjYtNDE3OC05MjVmLTc0MTgxYzk1MGQyOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyOVQyMTE2NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mOTA1MmE5Nzg2NGU2NGJjNzlmODA0NjNlNjNhMDMyZTc5M2NjYzgxZmNmZTJkNGIyZDhmYjNmOWMxYTFmN2ZjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.feakQYcaB0w9OSCt9Yx5b6zQgFh1VtjQesdd2gocDD4)

I don't see that error when I open it. Mine just doesn't load the Webcontainer (on classic editor).
![image](https://github.com/stackblitz/core/assets/76132013/55519224-da4d-47c3-9a05-3182cde39099)

**Comment by coder0107git** [comment]
On the new editor it only half loads for me (e.g. no syntax highlighting).

![image](https://github.com/stackblitz/core/assets/76132013/adc8a89e-f41f-491d-936d-b384ba0a833e)

**Comment by bcrhbrhcdb** [comment]
I kinda need the new editor... How would I fix it?

**Comment by bcrhbrhcdb** [comment]
How would i go to api.github.com?

**Comment by coder0107git** [comment]
> Anyone gonna update on this issue?

As soon as anyone has any additional information to contribute.

**Comment by coder0107git** [comment]
I was just looking at this again and found that [`w-credentialless-staticblitz.com`](https://w-credentialless-staticblitz.com/) is blocked on the computer that it's not working on (flagged as a parked domain)

**Comment by coder0107git** [comment]
> I kinda need the new editor... How would I fix it?

Any link that starts with `stackblitz.com/~/github.com/` is the new editor.

> How would i go to api.github.com?

`api.github.com` is not meant for users to access. It's meant to provide a way for apps and services to access and interact with Github.

**Comment by bcrhbrhcdb** [comment]
Yes its blocked

**Comment by bcrhbrhcdb** [comment]
Any updates?

**Comment by coder0107git** [comment]
> Any updates?

Oops. I was meaning to follow up and forgot. 🤦


> Found a fix: Get a package-lock, package.json, and git files.

Great that you found a fix! What do you mean by "Get a package-lock, package.json, and git files"? Add them to my repo? My repo already has a `pnpm-lock.yaml` and `package.json` file so I don't know what good readding pre-existing files would be.

**Comment by bcrhbrhcdb** [comment]
I added package.json, package-lock.json, and the git file.

**Comment by bcrhbrhcdb** [comment]
It was probably because [w-credentialless-staticblitz.com](https://w-credentialless-staticblitz.com/) was blocked

**Comment by coder0107git** [comment]
> Yes, and it is still blocked.

And StackBlitz works for you now by adding those files?  On Webcontainer or Engine block?

**Comment by bcrhbrhcdb** [comment]
The first time it worked for me I created a stack blitz starter, and then connected it thru github. So I assumed that I needed those files. I assume webcontainer
